--- gitui-0.27.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ gitui-0.27.0/Cargo.toml	2025-04-14T12:47:50.533433+00:00
@@ -131,7 +131,7 @@
 version = "1"
 
 [dependencies.parking_lot_core]
-version = "=0.9.10"
+version = "0.9.10"
 
 [dependencies.ratatui]
 version = "0.29"
@@ -145,7 +145,7 @@
 version = "1.12"
 
 [dependencies.ron]
-version = "0.8"
+version = "0.9"
 
 [dependencies.scopeguard]
 version = "1.2"
@@ -187,7 +187,7 @@
 version = "1.12"
 
 [dependencies.unicode-truncate]
-version = "2.0"
+version = ">=1.0,<3.0"
 
 [dependencies.unicode-width]
 version = "0.2"
@@ -214,7 +214,6 @@
     "ghemoji",
     "regex-fancy",
     "trace-libgit",
-    "vendor-openssl",
 ]
 ghemoji = ["gh-emoji"]
 regex-fancy = [
@@ -227,7 +226,6 @@
 ]
 timing = ["scopetime/enabled"]
 trace-libgit = ["asyncgit/trace-libgit"]
-vendor-openssl = ["asyncgit/vendor-openssl"]
 
 [badges.maintenance]
 status = "actively-developed"
