--- notify-debouncer-mini-0.6.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ notify-debouncer-mini-0.6.0/Cargo.toml	2025-01-25T07:02:57.423796+00:00
@@ -49,8 +49,6 @@
     "dep:crossbeam-channel",
     "notify/crossbeam-channel",
 ]
-default = ["macos_fsevent"]
-macos_fsevent = ["notify/macos_fsevent"]
-macos_kqueue = ["notify/macos_kqueue"]
+default = []
 serde = ["notify-types/serde"]
 serialization-compat-6 = ["notify/serialization-compat-6"]
